web/web.go

```diff
@@ -67,6 +68,29 @@ import (
 
 var localhostRepresentations = []string{"127.0.0.1", "localhost"}
 
+var (
+	requestDuration = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "http_request_duration_seconds",
+			Help:    "Histogram of latencies for HTTP requests.",
+			Buckets: []float64{.05, .1, .25, .5, 1, 2.5, 5, 10},
+		},
+		[]string{"handler"},
+	)
+	responseSize = prometheus.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "http_response_size_bytes",
+			Help:    "Histogram of response size for HTTP requests.",
+			Buckets: prometheus.ExponentialBuckets(200, 3.0, 8),
```

Comment:
A 100MB response is perfectly possible, I'd go for bigger jumps here and aim to top out around 1GB